app/assets/images/layout/brand.svg

```diff
@@ -1,3 +1 @@
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="60" height="14" viewBox="-253.3 389.5 60.7 14"><path fill="#FFF" d="M-243 399.9h-1.4l-1-7.7-2.7 5.3-2.8-5.3-1 7.7h-1.4l1.4-10.2h1.2l2.7 5 2.6-5h1.2l1.2 10.2zM-235.9 399l-.2 1h-.1c-.6 0-1-.1-1.2-.2-.2-.2-.4-.4-.6-.8-.6.7-1.3 1-2.2 1-.7 0-1.2-.2-1.6-.6-.4-.4-.6-.9-.6-1.5 0-.5.1-.9.4-1.2.3-.3.6-.6 1.1-.8.4-.2.9-.3 1.3-.4.5-.1.9-.1 1.4-.1 0-.6-.1-1.1-.4-1.3-.2-.2-.6-.3-1.2-.3s-1.2.2-1.9.6l-.2-1.1c.6-.5 1.4-.7 2.4-.7.9 0 1.5.3 1.9.8.4.5.6 1.4.6 2.7v1.5c0 .6 0 1 .1 1.2.1.2.3.3.7.3l.3-.1zm-2.3-1v-1.7c-1 0-1.8.2-2.2.4-.4.2-.7.6-.7 1.1 0 .7.4 1.1 1.1 1.1.3 0 .7-.1 1-.3.3-.1.6-.3.8-.6zM-228.8 399.9h-1.3V396c0-.8-.1-1.3-.4-1.7-.3-.3-.7-.5-1.4-.5-.9 0-1.6.3-1.9.9v5.2h-1.3v-7.1h1l.1.8c.7-.6 1.4-.9 2.3-.9 1.9 0 2.9 1.1 2.9 3.3v3.9zM-221.5 399l-.2 1h-.1c-.6 0-1-.1-1.2-.2-.2-.2-.4-.4-.6-.8-.6.7-1.3 1-2.2 1-.7 0-1.2-.2-1.6-.6-.4-.4-.6-.9-.6-1.5 0-.5.1-.9.4-1.2.3-.3.6-.6 1.1-.8.4-.2.9-.3 1.3-.4.5-.1.9-.1 1.4-.1 0-.6-.1-1.1-.4-1.3-.2-.2-.6-.3-1.2-.3s-1.2.2-1.9.6l-.2-1.1c.6-.5 1.4-.7 2.4-.7.9 0 1.5.3 1.9.8.4.5.6 1.4.6 2.7v1.5c0 .6 0 1 .1 1.2.1.2.3.3.7.3l.3-.1zm-2.2-1v-1.7c-1 0-1.8.2-2.2.4-.4.2-.7.6-.7 1.1 0 .7.4 1.1 1.1 1.1.3 0 .7-.1 1-.3.3-.1.6-.3.8-.6zM-217.9 403.5c-1.1 0-1.9-.2-2.5-.7-.5-.4-.8-.9-.8-1.5 0-.8.4-1.3 1.3-1.6-.5-.2-.7-.5-.7-.9 0-.2.1-.3.2-.5s.4-.4.6-.5c-.8-.6-1.2-1.4-1.2-2.3 0-.8.3-1.5.8-2s1.2-.8 2.1-.8c.6 0 1.2.2 1.8.5.5-.3 1.1-.4 1.7-.4h.1l-.2.9h-.3c-.3 0-.6 0-.8.1.4.5.6 1.1.6 1.7 0 .8-.3 1.4-.8 2-.5.6-1.2.8-2.1.8-.4 0-.7 0-1-.1-.2.1-.3.3-.3.4 0 .2.1.3.4.4.3.1.9.2 1.8.4.3 0 .5.1.7.1.2 0 .5.1.7.2.3.1.5.2.7.3.2.1.4.3.5.5.1.2.2.5.2.7 0 .6-.3 1.1-1 1.6-.5.5-1.4.7-2.5.7zm-.2-9.8c-.5 0-.9.2-1.2.5-.3.3-.4.7-.4 1.3 0 .6.2 1 .5 1.3.3.3.7.5 1.2.5s.9-.2 1.2-.5c.3-.3.4-.8.4-1.3s-.2-1-.5-1.3c-.3-.4-.7-.5-1.2-.5zm.6 6.5c-.2 0-.6-.1-1.3-.2-.3.1-.6.3-.8.5-.2.2-.3.5-.3.7 0 .3.2.6.6.8.4.2.9.3 1.6.3.7 0 1.2-.1 1.6-.3.4-.2.6-.5.6-.8-.1-.4-.8-.7-2-1zM-208 396.3h-5.1c.2 1.7 1 2.5 2.7 2.5.7 0 1.4-.1 2-.4l.2 1c-.8.4-1.6.6-2.4.6-1.2 0-2.1-.4-2.8-1.1-.7-.7-1-1.6-1-2.7 0-1 .3-1.9.9-2.6.6-.7 1.5-1 2.5-1 .9 0 1.6.3 2.2.8.6.5.9 1.3.9 2.2 0 .3 0 .5-.1.7zm-3-2.5c-1 0-1.7.5-2 1.6h3.7c0-.5-.1-.9-.4-1.2-.3-.3-.7-.4-1.3-.4zM-203.7 399.9h-1.8v-10.2h1.8v10.2zM-192.7 398.7l-1.1 1.3-1-.8c-.8.6-1.8.9-2.9.9-1.5 0-2.7-.5-3.6-1.5-.9-1-1.4-2.3-1.4-3.8 0-1.6.5-2.9 1.4-3.8.9-1 2.1-1.5 3.6-1.5s2.7.5 3.6 1.5c.9 1 1.4 2.2 1.4 3.8 0 1.2-.3 2.3-.9 3.2l.9.7zm-2.7-6.5c-.5-.6-1.2-1-2.2-1-1 0-1.7.3-2.2 1-.5.6-.8 1.5-.8 2.5 0 1.1.3 1.9.8 2.5.5.6 1.2 1 2.2 1 .5 0 1-.1 1.4-.3l-1.2-1 1.1-1.3 1.3 1.1c.3-.5.4-1.2.4-1.9-.1-1.1-.3-1.9-.8-2.6z"/></svg>
```

Comment:
Is this change related to the thing you're describing in the PR? If not, please create a separate one.

Reply:
Not relevant. I reverted the change